fix(html5shiv): provide a https source for the script

stexandev opened the pull request:
loading the script from a http source violates current security best practices

closes #1

stexandev committed:
fix(html5shiv): provide a https source for the script

loading the script from a http source violates current security best practices

closes #1

ubbo commented:
looks good to me. only question is if the whole ie<9 block still makes sense, as our servers are not reachable via tls for ie<11 anymore if i am not mistaken (according to https://wiki.mozilla.org/Security/Server_Side_TLS).

stexandev committed:
fix(html5shiv): drop support for ie6-8

we have decided to drop the support rather than to provide an
updated js source

closes #1

fungunga reviewed (approved):
Jes, seems fine to me.
